php/Terminus/Models/Environment.php

```diff
@@ -722,6 +722,34 @@ function ($param) {
         return $workflow;
     }
 
+    /**
+     * Remove a HTTPS certificate from the environment
+     *
+     * @return array $workflow
+     *
+     * @throws \Terminus\Exceptions\TerminusException
+     */
+    public function disableHttpsCertificate()
+    {
+        if (!$this->settings('ssl_enabled')) {
+            throw new TerminusException('The {env} environment does not have https enabled.', ['env' => $this->id]);
+        }
+        $response = $this->request->request(
+            "sites/{$this->site->id}/environments/{$this->id}/settings",
+            [
+                'method' => 'put',
+                'form_params' => [
+                    'ssl_enabled' => false,
+                    'dedicated_ip' => false,
+                ],
+            ]
+        );
+        if ($response['status_code'] !== 200) {
```

Comment:
I think Guzzle should throw an error if the status code isn't 200. What error does it naturally give you?